File: Program diferents lenguages/Abstract factory/AbstractFactoryJava/src/MercedesDealer.java
public  class MercedesDealer extends AbstractFactory
    {
        @Override
        public  CAR createCAR()
        {
            return new MercedesCAR();
        }


        @Override
        public  TRUCK createTRUCK()
        {
            return new MercedesTRUCK();
        }


    }

File: Program diferents lenguages/Abstract factory/AbstractFactoryc#/AbstractFactory/AbstractFactory/MercedesDealer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AbstractFactory
{
   public  class MercedesDealer : AbstractFactory
    {

        public override CAR createCAR()
        {
            return new MercedesCAR();
        }



        public override TRUCK createTRUCK()
        {
            return new MercedesTRUCK();
        }


    }
}
